rmgpy/tools/simulate.py

```diff
@@ -139,7 +138,7 @@ def simulate(rmg, diffusionLimited=True):
         if reactionSystem.sensitiveSpecies:
             plotSensitivity(rmg.outputDirectory, index, reactionSystem.sensitiveSpecies)
 
-def runSensitivity(inputFile, chemkinFile, dictFile, diffusionLimited=True):
+def runSimulation(inputFile, chemkinFile, dictFile, diffusionLimited=True):
```

Comment:
If you're renaming this anyway, how about switching to lowercase with underscores? (And the other methods as well.)

Reply:
Is it more important to have consistent style within RMG or to comply with PEP8? I would think the former, but I presume all of you have had discussions about this already, so I'm happy to make the change.

Reply:
I think we made the decision to implement future code using PEP8, and any changes to old code along the way as we can.

Reply:
Should I also rename `simulateTest.py` to `simulate_test.py` or something similar?

Reply:
Hmm, I don't think we considered that previously... The [style guide](https://github.com/ReactionMechanismGenerator/RMG-Py/wiki/Programming-Style-Guide) says to use `simulateTest.py` because that's how we've been doing it, but it doesn't make much sense.

I think you can keep it as `simulateTest.py` for now, but we might consider renaming all our test files in the future.